collections/_article/dev-snapshot-godot-4-3-beta-1.md

```diff
@@ -0,0 +1,255 @@
+
+#### TileMap layers as nodes
+TileMap layers are now exposed as individual TileMapLayer nodes ([GH-89179](https://github.com/godotengine/godot/pull/89179)) , which means less clutter in the inspector, a simpler API, and is also more in line with common Godot design patterns.
+
+To avoid the small drawbacks that would come with that change, we added new editor features, for example the ability to select all layers in the currently edited scene. The TileMap node itself is marked as deprecated but will stay for a while (it will not get any new features though).
+
+To help with the transition, you can automatically transform a TileMap node to a set of TileMapLayer nodes via a dropdown menu entry in the editor. You’ll have to update your scripts, but don’t worry, the API is very similar.
+
+### Editor
+
+#### Fixes for invalid/corrupt scenes
+A common complaint among users is that they get locked out of editing a scene because it contained an instance of a different scene that no longer exists (due to being renamed or deleted). Users learned to dread the well-known “Scene invalid/corrupt” error that resulted from this. ([GH-86781](https://github.com/godotengine/godot/pull/86781)) aims to improve the situation by allowing you to open, edit, and fix scenes that have been corrupted due to a missing dependency. This should make the process of refactoring your projects within Godot feel quite a bit safer.
+
+Another bugfix on the GDScript side may also help solve situations where using preload() with cyclic dependencies leads to scenes being flagged as invalid ([GH-85501](https://github.com/godotengine/godot/pull/85501)).
+
+Loading of scenes with corrupted or missing dependencies will no longer be aborted ([GH-85159](https://github.com/godotengine/godot/pull/85159)), allowing you to use and fix such scenes without external tools.
+
+#### Add ufbx for importing .fbx files without FBX2glTF
+This monumental effort by Samuli Raivio and Ernest Lee  incorporates the popular ufbx library into Godot to allow for seamlessly importing .fbx files ([GH-81746](https://github.com/godotengine/godot/pull/81746)). Previously users would have to download the FBX2glTF tool separately and Godot would invoke it to convert .fbx files to glTF files in order to import them. ufbx allows us to avoid this process and import .fbx files directly. Although the ufbx library has gone through extensive rigorous testing, this is a huge change and the .fbx file format is notoriously difficult to work with, so please test this carefully and report any bugs you find.
```

Comment:
```suggestion
This monumental effort by Samuli Raivio and Ernest Lee incorporates the popular ufbx library into Godot to allow for seamlessly importing .fbx files ([GH-81746](https://github.com/godotengine/godot/pull/81746)). Previously users would have to download the FBX2glTF tool separately and Godot would invoke it to convert .fbx files to glTF files in order to import them. ufbx allows us to avoid this process and import .fbx files directly. Although the ufbx library has gone through extensive rigorous testing, this is a huge change and the .fbx file format is notoriously difficult to work with, so please test this carefully and report any bugs you find.
```